### DISCUSSION TITLE
Rxjs/Angular support?

### DISCUSSION THREAD

**Discussion by NateRadebaugh** [TanStack/table#4226] [Ideas]
With this now being framework agnostic I thought I'd see an example for using with angular apps. Would this be trivial to set up, or do observables and rxjs make it more complex than the other supported frameworks?

Thanks!

**Comment by KevinVandy** [comment]
No one has made a PR for an Angular adapter yet, but it would be welcomed

**Reply by jrgokavalsa** [reply]
Hi KevinVandy,

I have created an angular adapter for the library and deployed a few examples at the following page: https://jrgokavalsa.github.io/angular-tanstack-table/  I would appreciate some help to make a PR request to the main repository. If you are interested, please check out the code and let me know if you have any feedback or suggestions. Thank you for your time and attention.

Link updated: https://jrgokavalsa.github.io/angular-tanstack-adapter/

**Reply by KevinVandy** [reply]
yes we're interested! I might bring @crutchcorn in on this.

**Reply by KevinVandy** [reply]
@jrgokavalsa Can you make your link available though?

**Reply by crutchcorn** [reply]
Happy to PR review 😊

**Reply by jrgokavalsa** [reply]
@KevinVandy @crutchcorn 

Thanks for the reply, I have made some updates and the link to latest version https://jrgokavalsa.github.io/angular-tanstack-adapter/ .  Please let me know if you have any questions or feedback.

**Reply by KevinVandy** [reply]
@jrgokavalsa Would you like to make this into a PR?

**Reply by jrgokavalsa** [reply]
That would be great @KevinVandy  😊

**Comment by jrgokavalsa** [comment]
@KevinVandy @crutchcorn 

Need help with inital steps to create a library with angular in the project

I have implemented angular-table package manually, the build gets successful if I remove **export * from './flexrender.directive'**, I cannot import flexrender directive when i use the build in another project. If i include the line getting below error

Add @babel/plugin-proposal-decorators (https://github.com/babel/babel/tree/main/packages/babel-plugin-proposal-decorators) to the 'plugins' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-decorators (https://github.com/babel/babel/tree/main/packages/babel-plugin-syntax-decorators) to the 'plugins' section to enable parsing.
src/flexrender.directive.ts (9:0)

If include the plugins in packagejson file the error is gone but i cannot use flexrender directive in another project.

Error: Flexrender directive must be standlone.

You can check the code at
https://github.com/jrgokavalsa/table/tree/feat-angular-adapter

**Comment by jrgokavalsa** [comment]
@KevinVandy @crutchcorn

Pull request at https://github.com/TanStack/table/pull/5326
